### DISCUSSION TITLE
Automatic FONT setting on GSE-Generator

### DISCUSSION THREAD

**Discussion by frost988** [brunolee-GIT/GSE-Generator#7] [Q&A] [answered]
Someone have an idea how could I modify the GSE-Generator to use my own "configs.overlay.ini" and the "fonts" folder inside the steam_settings? Probably modifying the .bat file but I have zero experience with coding... I wish I can do it, I found a cool font so I'd like to set this to every single emu settings that the generator generates :). That's an amazing tool, I hope we have a big community.
Cheers guys!

**Comment by brunolee-GIT** [comment, marked as answer]
Extract [GSE_Generator_fonts_update](https://github.com/user-attachments/files/16398272/GSE_Generator_fonts_update.zip) directly to where the "**GSE_Generator.bat**" file is.

Place your favorite font files in the `"...\Tools\ACHIVEMENTS\fonts\"` folder.
Edit `"...\Tools\ACHIVEMENTS\configs.overlay.ini"` to your liking. (**there is no need if it is to change the font**)
***
**When running "GSE_Generator.bat":**
- If `"...\Tools\ACHIVEMENTS\configs.overlay.ini"` exists, it will use this one, or he will create what already create before.
- If exist font files in `"...\Tools\ACHIVEMENTS\fonts\"` will open a font selection window.
>[!NOTE]
>>The selected font will be copied to the "fonts" folder in "steam_settings" and edited in the "configs.overlay.ini" file automatically.

**Comment by frost988** [comment]
Wow, what an amazing update! I was thinking about something basic but now it supports multi-fonts with a selector, damn!... Good job as always @brunolee-GIT. I really appreciate your tool , many thanks <3.
